pkg/daemon/config.go

```diff
@@ -32,6 +33,7 @@ type ptp4lConf struct {
 	sections     []ptp4lConfSection
 	mapping      []string
 	profile_name string
+	clock_type   event.ClockType
```

Comment:
ClockType isn't at all related to events, and this events code may be pulled out when upstreaming.  Can you move it to be defined somewhere else?  Maybe github.com/openshift/linuxptp-daemon/pkg/config

Reply:
Currently this is the only place we handle a ptp4l config update event, which is what we are using the compute clockType. Please suggest how we could decouple this logic from the existing implementation, and I will be happy to follow up and update this logic in a follow up PR.